bokeh/_version.py

```diff
@@ -40,7 +40,7 @@ def get_config():
     # _version.py
     cfg = VersioneerConfig()
     cfg.VCS = "git"
-    cfg.style = "git-describe"
+    cfg.style = "pep440"
```

Comment:
@mattpap our `setup.cfg` has "pep440" but this cfg is only consulted when versioneer is *installed* so evidently it was never actually used (until now, if we keep the change on the line above). Do you have an opinion? The difference between the two is outlined here:

https://github.com/warner/python-versioneer/blob/master/details.md#how-do-i-select-a-version-style

Reply:
I recall both styles being mixed in some situations in the past (e.g. bokehjs has `[+-]` in a regex to accommodate this). I don't have preference as long as things are consistent, though `git-describe` seems cleaner to me.

Reply:
OK I am happy to leave it `git-describe` I just want to make sure the `setup.cfg` matches our intentions. I will revert this and update the value the config.